crates/polars-io/src/csv/write.rs

```diff
@@ -53,10 +57,7 @@ where
     }
 }
 
-impl<W> CsvWriter<W>
```

Comment:
Can you revert this change? Seems like a style change.

Reply:
done. I saw this syntax being used in another source file and it looked more condense.
Hence I did this change.

Reverted for now.